csunplugged/tests/general/loaders/test_generalpages_loader.py

```diff
@@ -0,0 +1,145 @@
+from os.path import join, dirname
+from django.conf import settings
+from django.test import override_settings
+from tests.BaseTestWithDB import BaseTestWithDB
+from general.models import GeneralPage
+from general.management.commands._GeneralPagesLoader import GeneralPagesLoader
+from utils.errors.CouldNotFindYAMLFileError import CouldNotFindYAMLFileError
+from utils.errors.InvalidYAMLValueError import InvalidYAMLValueError
+from utils.errors.MissingRequiredFieldError import MissingRequiredFieldError
+from utils.errors.EmptyYAMLFileError import EmptyYAMLFileError
+
+BASE_PATH = "tests/general/loaders/assets/"
+test_template_settings = settings.TEMPLATES
+default_path = test_template_settings[0]["DIRS"][0]
+new_path = join(dirname(default_path), BASE_PATH, "templates/")
+test_template_settings[0]["DIRS"].append(new_path)
+url_conf = join(BASE_PATH, "urls").replace("/", ".")
+
+
+@override_settings(TEMPLATES=test_template_settings)
+@override_settings(ROOT_URLCONF=url_conf)
+class GeneralPageLoaderTest(BaseTestWithDB):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.loader_name = "generalpage"
+        self.BASE_PATH = join(BASE_PATH, "generalpages/")
+
+    def test_basic_generalpage_loader_config(self):
```

Comment:
is there a reason `general_page` has been concatenated into one?

Reply:
As this is the model `GeneralPage` converted to lower case, rather than the text "General Page" converted to lower case. However now looking it appears other tests use text rather than the explicit model name. Do you prefer text over model name?

Reply:
prefer text